var/spack/repos/builtin/packages/py-onnxconverter-common/package.py

```diff
@@ -12,7 +12,7 @@ class PyOnnxconverterCommon(PythonPackage):
     homepage = "https://github.com/microsoft/onnxconverter-common"
     url      = "https://github.com/microsoft/onnxconverter-common/archive/refs/tags/v1.9.0.tar.gz"
 
-    version('1.9.0', sha256='e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855')
+    version('1.9.0', sha256='32315bcc844a8203092f3117a4a092ac6cf03d6a20145477e284f1172557d6f9')
```

Comment:
Why the checksum change?

Reply:
No idea why this happened. I don't see any signs of release being republished.

Reply:
Can you open an issue with the developers and ask?

Reply:
Did you get a chance to do this? We don't want to allow checksum changes for no reason because it could imply that a release was hacked.

Reply:
It's not a checksum for the zip file, or a PyPI file instead, is it? That's another possibility. I would like to wait up to a week to get a response from the developers.

Reply:
Nope. Sure, let's wait.

Reply:
So, just for fun ran `grep e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` on my local repo, and - suddenly - found that tarball for `madgraph5amc@2.8.0` had the same hash. I didn't touch that package for a long time, so not sure how that hash got into that file.

Reply:
Well, we never got a response from the developers, so I'm going to assume they weren't hacked.